rke/k8s_version_info.go

```diff
@@ -643,6 +643,10 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.6.7-patch0",
 			MinRKEVersion:     "1.3.13-rc0",
 		},
+		"v1.25": {
+			MinRancherVersion: "2.7.1-patch0",
```

Comment:
Why is the `MinRancherVersion` a patch0?

Reply:
We've occasionally tagged `-patch` versions after Rancher releases for specific bug fixes. Kubernetes versions should still load for them same as `v2.7.1` so we set it as min version beforehand than having to make a change later on.